eng/pipelines/coreclr/templates/build-job.yml

```diff
@@ -210,7 +216,7 @@ jobs:
         displayName: Run CoreCLR Tools unit tests
 
     # Build native test components
-    - ${{ if ne(parameters.isOfficialBuild, true) }}:
+    - ${{ if and(ne(parameters.isOfficialBuild, true), ne(parameters.archType, 'armel')) }}:
```

Comment:
Should we instead introduce a boolean parameter to the template to disable all the test steps rather than conditioning for the archType directly?

Reply:
Thank you. I updated.

Reply:
Thank you. I updated.

Reply:
@safern Is it wrong? Could you please review it? Thank you.

Reply:
You need to pass down that parameter on `platform-matrix` when in armel and set it to true. That way it will work, at the moment it is just taking the parameter default value to false.

Reply:
I added ```disableClrTest``` parameter and set it to true in ```platform-matrix```. Is it the right way?